include/model.php

```diff
@@ -12,7 +12,7 @@ class PLL_Model {
 	/**
 	 * Internal non persistent cache object.
 	 *
-	 * @var PLL_Cache
+	 * @var PLL_Cache<mixed>
```

Comment:
```suggestion
	 * @var PLL_Cache<PLL_Language|array>
```
As of today, `PLL_Model` caches only this two kind of data IMHO.

Reply:
It can also be `array<string>` because of [this](https://github.com/polylang/polylang/blob/46e1a5be8c95ddf7043b964de29ffa9b13fc4a1e/include/translated-post.php#L150) and [this](https://github.com/polylang/polylang/blob/46e1a5be8c95ddf7043b964de29ffa9b13fc4a1e/include/translated-term.php#L218).
The full type would be `PLL_Cache<PLL_Language|list<PLL_Language>|array<string>>` then. But then it triggers new issues like `Cannot call method get_tax_prop() on PLL_Language|string.` and `Cannot access property $is_default on PLL_Language|string.` and I don't see a way to fix this because the type returned by `PLL_Cache::get()` depends on the 1st argument passed to it (BTW, it might come back to bite our ass someday if we store all kinds of things in a same instance of `PLL_Cache` like in this case 🤔).

Reply:
Yeah you're right, it becomes too hard to maintain that way.